no module named mock

cloned from  github then ran `pip install -r requirements.txt`

```
$ python manage.py test

Creating test database for alias 'default'...
.......................................................................................................F....................EEE
======================================================================
ERROR: allauth.socialaccount.providers.openid.tests (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: allauth.socialaccount.providers.openid.tests
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/home/matt/django-allauth/allauth/socialaccount/providers/openid/tests.py", line 4, in <module>
    from unittest.mock import Mock, patch
ImportError: No module named mock


======================================================================
ERROR: allauth.socialaccount.providers.facebook.tests (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: allauth.socialaccount.providers.facebook.tests
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/home/matt/django-allauth/allauth/socialaccount/providers/facebook/tests.py", line 4, in <module>
    from unittest.mock import patch
ImportError: No module named mock


======================================================================
ERROR: allauth.socialaccount.providers.persona.tests (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: allauth.socialaccount.providers.persona.tests
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/home/matt/django-allauth/allauth/socialaccount/providers/persona/tests.py", line 4, in <module>
    from unittest.mock import patch
ImportError: No module named mock


======================================================================
FAIL: test_password_forgotten_url_protocol (allauth.account.tests.AccountTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/matt/django-allauth/allauth/account/tests.py", line 189, in test_password_forgotten_url_protocol
    self.assertTemplateUsed(resp, 'account/password_reset_from_key.html')
  File "/home/matt/conf/venvs/allauth/local/lib/python2.7/site-packages/django/test/testcases.py", line 538, in assertTemplateUsed
    self.fail(msg_prefix + "No templates used to render the response")
AssertionError: No templates used to render the response

----------------------------------------------------------------------
Ran 127 tests in 12.967s

FAILED (failures=1, errors=3)
```

The errors are from not having mock installed and the failure I filed as #837.

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,5 @@
 django>=1.4.3
+mock>=1.0.1
 # There is no OpenID lib that support both Python 2 and 3 from a single-source base.
 # So we have to choose between:
 # python-openid>=2.2.5

